RockWeb/Blocks/Finance/TransactionEntry.ascx.cs

```diff
@@ -312,6 +324,55 @@ protected override void OnLoad( EventArgs e )
             pnlDupWarning.Visible = false;
             nbSaveAccount.Visible = false;
 
+            string accountParameterType = string.Empty;
+            using ( var rockContext = new RockContext() )
+            {
+                if ( !string.IsNullOrWhiteSpace( PageParameter( "account" ) ) )
+                {
+                    List<int> accountParameter = new List<int>();
+                    accountParameterType = "invalid";
+                    foreach ( string acctId in PageParameter( "account" ).Split( ',' ) )
+                    {
+                        int id = 0;
+                        if ( int.TryParse( acctId, out id ) )
+                        {
+                            accountParameter.Add( id );
+                        }
+                    }
+                    _parameterAccounts = new FinancialAccountService( rockContext ).Queryable()
+                        .Where( f =>
+                        accountParameter.Contains( f.Id ) &&
+                        f.IsActive &&
+                        (f.StartDate == null || f.StartDate <= RockDateTime.Today) &&
+                        (f.EndDate == null || f.EndDate >= RockDateTime.Today) ).ToList();
+                    if ( _parameterAccounts.Count > 0 )
+                    {
+                        accountParameterType = "valid";
+                    }
+                }
+                if ( !string.IsNullOrWhiteSpace( PageParameter( "glaccount" ) ) )
+                {
+                    List<string> glAccountParameter = PageParameter( "glaccount" ).Split( ',' ).ToList();
+                    _parameterAccounts.AddRange( new FinancialAccountService( rockContext ).Queryable()
+                        .Where( f =>
+                        glAccountParameter.Contains( f.GlCode ) &&
+                        f.IsActive &&
```

Comment:
I added that flag and have also added a Block Setting that will let you bypass if desired.  The primary use case of this is for some Missionary Accounts that we don't want displayed on the default Contribution page.